src/spectrecoff/Tree.fs

```diff
@@ -13,17 +13,28 @@ type GuideStyle =
 
 type TreeLayout =
     {  Sizing: SizingBehaviour
-       Guides: GuideStyle }
+       Guides: GuideStyle 
+       ForeGroundColor: Color Option
+       BackGroundColor: Color Option
+       Decoration: Decoration Option}
 
 let defaultTreeLayout: TreeLayout =
     {  Sizing = Collapse
-       Guides = SingleLine }
+       Guides = SingleLine
+       ForeGroundColor = Some calmColor
+       BackGroundColor = None
+       Decoration = None }
 
 let toOutputPayload tree =
     tree 
     :> Rendering.IRenderable 
     |> Renderable
 
+let private toNullable option = 
+    match option with 
+    | None -> new System.Nullable<_>()
+    | Some a -> new System.Nullable<_>(a)
```

Comment:
Well, what would Spectre else expect? There is, e.g., no transparent color afaik, so that is achieved in Spectre by null. However, if FSharp there is a better option :)

Y let's move it to a common place once we need it again. The code base is still so small that I'm sure we'll remember it's existence.

Reply:
Removed new, and braces.